Content.Server/Chemistry/EntitySystems/SolutionRandomFillSystem.cs

```diff
@@ -20,18 +20,27 @@ public override void Initialize()
         SubscribeLocalEvent<RandomFillSolutionComponent, MapInitEvent>(OnRandomSolutionFillMapInit);
     }
 
-    public void OnRandomSolutionFillMapInit(EntityUid uid, RandomFillSolutionComponent component, MapInitEvent args)
+    private void OnRandomSolutionFillMapInit(EntityUid uid, RandomFillSolutionComponent component, MapInitEvent args)
```

Comment:
I must have changed this to private at some point, it hasn't caused any issues so far but maybe I should revert it?

Reply:
I mean if your IDE didn't shout at you, then you're probably good